internal/repositories/channel_subscriptions.go

```diff
@@ -32,10 +32,14 @@ func (r *ChannelSubscriptionsRepository) RemoveList(groupId int64, userIds []int
 
 func (r *ChannelSubscriptionsRepository) GetUserIdsByChannelId(channelId int64) ([]int64, error) {
 	var userIds []int64
-	err := r.db.Select("user_id").Where("channel_id = ?", channelId).Find(&userIds).Error
+	err := r.db.Table("channel_subscriptions").Select("user_id").Where("channel_id = ?", channelId).Find(&userIds).Error
```

Comment:
Create method for `ChannelSubscripions` TableName() that returns "channel_subscriptions". And then you can use 
```suggestion
	err := r.db.Table(model.ChannelSubsriptions{}.TableName()).Select("user_id").Where("channel_id = ?", channelId).Find(&userIds).Error
```
To avoid a lot of literals usage